### PR TITLE
Remove some dead code

### PR DESCRIPTION
**Pull request by N-R-K**
No real change in functionality intended.

### CODE REVIEW

File: video/out/aspect.c [hunk 1]
```diff
@@ -31,8 +31,6 @@ static void aspect_calc_panscan(struct mp_vo_opts *opts,
                                 int window_w, int window_h, double monitor_par,
                                 int *out_w, int *out_h)
 {
-    w *= monitor_par;
```
Comment (kasper93) [inline video/out/aspect.c:34]: This is static function, you could remove also unused parameter.

Comment (N-R-K) [reply video/out/aspect.c:34]: Passing just the height here where all other params expect w+h pair is a bit weird. And it'll be less work/annoyance if `w` becomes actually used again.

**Review by Dudemanguy — APPROVED** [review]
LGTM. We'll just keep the extra `w` param for now I suppose.

File: video/out/aspect.c (hunk 1 above)
Comment (kasper93) [reply video/out/aspect.c:34]: Yes, I agree. I don't know why w is not used though. But I don't want to dig into it.

**Pull request merged**